srcpkgs/asciinema/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'asciinema'
 pkgname=asciinema
 version=2.0.2
-revision=3
+revision=4
 build_style=python3-module
-pycompile_module="asciinema"
 hostmakedepends="python3-setuptools"
 depends="$hostmakedepends"
+checkdepends="python3-pip"
```

Comment:
Adding `pip` to install checkdepends automatically is a bad idea, we'd try to look into the actual test dependencies listed in `setup.cfg` or whatever.

That said, I'm not sure anything is actually running

> Ran 0 tests in 0.000s

Reply:
Yeah, it's dumb, but without it the test blows up as it tries to pip install something. I'm not sure which is the less bad route, install pip and keep it in-line with what upstream expects, or fix it in a patch.

Reply:
OK, looks like I was being a tad hyperbolic. It looks like the test dependency is actually python3-nose and it only needs pip if you don't have that. Lets see if that fixes things.